### PR TITLE
Fix transfer qr scan and other outstanding bugs

### LINKED ISSUE #243: Dead Link: Global Hotspot Explorer
Sign into an account without Hotspots.

Tap Global Hotspot Explorer - nothing happens.

This should go to explorer.helium.com/coverage
IMG_6F21DE6EFEB7-1

### LINKED ISSUE #245: Scanning QR during transfer changes to payment

### PR DESCRIPTION
**Pull request by tyler-whitman**
- Scanning an address QR while creating a transfer will no longer change you to a payment
- Removed the unused search icon from the wallet view
- Fixed the global explorer coverage link on the empty hotspots view
- Made the pin entry press color more transparent
Simulator Screen Shot - iPhone 11 - 2021-02-19 at 22 22 01

closes #243 
closes #245

### CODE REVIEW

File: ios/Podfile.lock [hunk 1]
```diff
@@ -736,8 +736,8 @@ SPEC CHECKSUMS:
   Flipper-PeerTalk: 116d8f857dc6ef55c7a5a75ea3ceaafe878aadc9
   Flipper-RSocket: a3acb8812d6adf127deb0a5edae2793b97e6b641
   FlipperKit: ab353d41aea8aae2ea6daaf813e67496642f3d7d
-  Folly: aeb27d02cdff07ca01f8a8a5a6dd5bcaf6be6f70
-  glog: 2ad46e202fbaa5641fceb4b2af37dcd88fd8762d
+  Folly: b73c3869541e86821df3c387eb0af5f65addfab4
```
Comment (tyler-whitman) [inline ios/Podfile.lock:739]: I don't know why these keep changing..

Comment (matthewcarlreetz) [reply ios/Podfile.lock:739]: same

File: src/theme/theme.ts [hunk 2]
```diff
@@ -6,6 +6,7 @@ import { TextProps } from 'react-native'
 const palette = {
   black: '#000000',
   white: '#FFFFFF',
+  whiteTransparent: '#FFFFFF66',
```
Comment (matthewcarlreetz) [inline src/theme/theme.ts:9]: nice! Somehow I didn't know you could do this with the alpha value.

**Review by matthewcarlreetz — APPROVED** [review]
Looks good!

**Pull request merged**